README.md

```diff
@@ -2127,71 +2154,6 @@ all together when trying to assign value to your equity:
-    Is it outstanding shares or fully diluted?*
-  - *What convertible securities are outstanding (convertible notes, SAFEs, or warrants), and
-    how much dilution can I expect from their conversion?*
-- **Valuation**:
-  - *What did the last round value the company at (that is, what is the preferred share price
-    times the total outstanding shares?*
-  - *What is the most recent 409A valuation?
-    When was it done, and will it be done again soon?*
-  - *What exit valuation will need to be achieved before common stock has positive value (that
-    is, what are the liquidation overhangs)?*
-- **Stock options**:
-  - *Do you allow early exercise of my options?*
-  - *Am I required to exercise my options within 90 days after I leave or am terminated?
-    Does the company extend the exercise window of the options of employees that depart?*
-- **Vesting**:
-  - *Are all employees on the same vesting schedule?*
-  - *Is there any acceleration of my vesting if the company is acquired?*
-  - *Do you have a policy regarding follow-on stock grants?*
-  - *Does the company have any repurchase right to vested shares?*
-
-This information will help you consider the benefits and drawbacks of possible
-[exercise scenarios](#stock-option-scenarios).
-
-🔹 If you’re considering working for a startup, there are further questions to ask in
-order to assess the state of the company’s business and its plans.
-Before or when you’re getting an offer is the right time to do this.
-Startups are understandably careful about sharing financial information, so you may not
-get full answers to all of these, but you should at least ask:
-
-- *How much money has the company raised (including in how many rounds, and when)?*
-- *What did the last round value the company at?*
-- *What is the aggregate liquidation preference on top of the preferred stock?*
-  (This will tell you how much the company needs to sell for before the common stock—your
-  equity—is worth something in an exit.)
-- *Will the company likely raise more capital soon?*
-- *How long will the company’s current funding last?*
-  (This will likely be given at the current burn rate, or how quickly a company is spending
-  its funding, so will likely not include calculations for things like future employee
-  salaries.)
-- *What is the hiring plan?
-  (How many people over what time frame?)*
-- *What is the revenue now, if any?
-  What are the revenue goals/projections?*
-- *Where do you see this company in 1 year and 5 years, in terms of revenue, number of
-  employees, and market position?*
-
-There are several other
-[resources](https://blog.wealthfront.com/stock-options-14-crucial-questions/) with
-[more questions](http://www.inc.com/atish-davda/5-questions-you-should-ask-before-taking-a-start-up-job-offer.html)
-like this to consider.
-
-🚧 Summarize the best items in the links above.
-
 ### Stock option scenarios
 
 Stock options are an exercise in deciding when to exercise and when to sell, if you can.
```

Comment:
Rephrase. Cutesy, could confuse ("an exercise in deciding when to exercise"). "...and figuring out if and when you can sell."